src/test/scala/io/iohk/ethereum/vm/CallOpFixture.scala

```diff
@@ -191,7 +191,8 @@ class CallOpFixture(val config: EvmConfig, val startState: MockWorldState) {
       inOffset: UInt256 = UInt256.Zero,
       inSize: UInt256 = inputData.size,
       outOffset: UInt256 = inputData.size,
-      outSize: UInt256 = inputData.size / 2
+      outSize: UInt256 = inputData.size / 2,
+      toAccessed: Boolean = false
```

Comment:
Actually, I had to look into the code to understand what this field was about.
I would have prefered something more descriptive like `addAddressToAccessList` or `keepTrackOfAccessAddresses`, but I tend to belong to the long-name-team.

Reply:
WDYT about `toAlreadyAccessed`?

Reply:
it seems clearer for me.